autowebcompat/network.py

```diff
@@ -155,7 +155,7 @@ def accuracy(y_true, y_pred):
     return K.mean(K.equal(y_true, K.cast(y_pred < 0.5, y_true.dtype)))
 
 
-def compile(model, optimizer='sgd', loss_func=contrastive_loss):
+def compile(model, optimizer, loss_func=contrastive_loss):
```

Comment:
Add an assertion at the beginning that the optimizer is in the list of supported optimizers. Let's leave the default as it was.

Reply:
same with this case also